src/challenges.js

```diff
@@ -1,49 +1,131 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(value1, value2) {
+  if (value1 === true && value2 === true) {
+    return true;
+  }
+  return false;
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, height) {
+  let calc = (base * height) / 2;
+  return calc;
 }
 
 // Desafio 3
-function splitSentence() {
-  // seu código aqui
+function splitSentence(str) {
+  return str.split(' ');
 }
 
 // Desafio 4
-function concatName() {
-  // seu código aqui
+function concatName(array) {
+  for (const key in array) {
+    return array[array.length - 1] + ', ' + array[0];
+  }
 }
 
 // Desafio 5
-function footballPoints() {
-  // seu código aqui
+function footballPoints(wins, ties) {
+  return (wins * 3) + (ties);
 }
 
 // Desafio 6
-function highestCount() {
-  // seu código aqui
+function highestCount(array) {
+  let hightNumber = array[0];
+  let count = 0;
+  for (let index = 0; index < array.length; index += 1) {
+    if (array[index] >= hightNumber) {
+      hightNumber = array[index];
+    }
+  }
+  for (let index = 0; index < array.length; index += 1) {
+    if (array[index] === hightNumber) {
+      count += 1;
+    }
+  }
```

Comment:
```suggestion
  for (let index = 0; index < array.length; index += 1) {
    if (array[index] >= hightNumber) {
      hightNumber = array[index];
    }
    if (array[index] === hightNumber) {
      count += 1;
    }
  }
```
fui no plantao do sumo e desse jeito aí você está deixando o 2 if dentro do for e não precisa usar dois for, e diminui assim a complexidade.